package.json

```diff
@@ -17,6 +17,7 @@
     "lint": "eslint lib/ test/",
     "lint:fix": "eslint --fix lib/ test/",
     "build": "parcel build lib/index.js --no-source-maps --out-file cf-extension-api.js --global contentfulExtension",
+    "build:debug": "parcel build lib/index.js --out-file cf-extension-api.js --global contentfulExtension",
```

Comment:
This allows source maps which allows for placing inline debuggers in the iframe. Pretty useful.